scripts/Setup-DevEnvironment.ps1

```diff
@@ -0,0 +1,2 @@
+. "$PSScriptRoot/Setup-Submodule.ps1"
```

Comment:
nit: do you think may be we should call this file `Setup-LocalDevEnvironment.ps1`? not a big deal, to me this file name sounds like setting up the "DEV" environment, maybe its just me.

Reply:
Makes sense. I chose this name initially because I'm mirroring the name of [the same file in `NuGetGallery`](https://github.com/NuGet/NuGetGallery/blob/master/tools/Setup-DevEnvironment.ps1)